tests/test_rewards.py

```diff
@@ -82,7 +82,13 @@ def test_accuracy_reward_wrong_answer(self):
         """Test accuracy_reward with an incorrect answer."""
         completion = [[{"content": r"\boxed{\frac{64}{400}}"}]]
         solution = [r"\frac{63}{400}"]
+        rewards = accuracy_reward(completion, solution)
+        self.assertEqual(rewards[0], 0.0)
 
+    def test_accuracy_reward_wrong_answer_no_latex(self):
```

Comment:
This test fails on `main` because we defaulted to reward = 1 when the gold answer could not be parsed.